### ISSUE TITLE
TypeDefs merged incorrectly when typePaths is not set

### ISSUE THREAD

**Issue by abogaart** [nestjs/graphql#133]
## I'm submitting a...

<pre><code>
[x] Regression 
[ ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When configuring a GraphQLModule with the property typeDefs set, but without the property typePaths set, the typeDefs will be incorrectly merged.

This used to work in 5.5.3 and seems to be introduced [here](https://github.com/nestjs/graphql/pull/127/commits/59a851efabf9a898642698c2d83cc171fb7cb2a4)

Previously, not having `typePaths` set would initialize `const typeDefs` as an empty array. After the change in #127, it is initialized as null and the null value changes the behavior of [./utils/extend.util.extend()](https://github.com/nestjs/graphql/blob/41028692708145bbc524f49610bf7f7d4a61303c/lib/utils/extend.util.ts#L3), which identifies the null value of `obj1` as an object and will try to spread the string or array value of `obj2` [here](https://github.com/nestjs/graphql/blob/41028692708145bbc524f49610bf7f7d4a61303c/lib/utils/extend.util.ts#L14).

## Expected behavior

Using GraphQLModule with typeDefs but without typePaths should work fine.

## Minimal reproduction of the problem with instructions

- `nest new fix-merged-typedefs`
- `cd fix-merged-typedefs`
- `yarn add @nestjs/graphql graphql apollo-server-express`
- edit `./src/app.module.ts`

```
import { Module } from '@nestjs/common';
import { GraphQLModule } from '@nestjs/graphql';

@Module({
  imports: [
    GraphQLModule.forRoot({
      typeDefs: `
      type User {
        id: ID!
      }
      `,
    }),
  ],
})
export class AppModule {}
```
- `yarn run start`
- output in terminal:
```
Syntax Error: Unexpected [

GraphQL request (2:9)
1:
2:         [object Object]
```

## Environment

<pre><code>
Nest version: 5.7.3

 
For Tooling issues:
- Node version: 10.13.0  
- Platform: Mac 

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Thanks for reporting. Should be fixed in the `@nestjs/graphql@5.5.5`. Let me know if you face any issue.

**Comment by abogaart** [comment]
That's fast! Works like a charm, thanks!
